src/types/reactivity.ts

```diff
@@ -0,0 +1 @@
+export type Getter<T> = () => T
```

Comment:
Are we not at the appropriate Vue version for this export?

Reply:
I'm not sure Getter is actually defined in the latest vue. I see [MaybeRefOrGetter](https://vuejs.org/api/utility-types.html#maybereforgetter) of course, but don't see Getter.

Reply:
Oh weird ok